dev/NavigationView/NavigationView.cpp

```diff
@@ -1905,6 +1905,11 @@ void NavigationView::ChangeSelection(const winrt::IInspectable& prevItem, const
         ChangeSelectStatusForItem(nextItem, true /*selected*/);
         RaiseSelectionChangedEvent(nextItem, isSettingsItem, recommendedDirection);
         AnimateSelectionChanged(nextItem);
+
+        if (auto nvi = NavigationViewItemOrSettingsContentFromData(nextItem))
```

Comment:
Use `const` here please
```suggestion
        if (auto const nvi = NavigationViewItemOrSettingsContentFromData(nextItem))
```

Reply:
What's the recommendation here? When next to all calls to this method in the same code file use the same pattern but no 'const', should new code use 'const' then? Looked at existing code here and thus would be good to know for the future. I personally would say 'use const' but there might be an argument to make for code file precedence.

Reply:
Guidance is use const where possible. Old code might not do it, but new code should use it wherever possible. @StephenLPeters please correct me if I am mistaken.

Reply:
Here a comment from a previos PR: https://github.com/microsoft/microsoft-ui-xaml/pull/2051#discussion_r387181219